app/assets/javascripts/lookup_keys.js

```diff
@@ -140,7 +140,7 @@ function undo_remove_child_node(item){
 function toggleOverrideValue(item) {
   var override = $(item).is(':checked');
   var fields = $(item).closest('.fields');
-  var fields_to_disable = fields.find("[name$='[required]'],[id$='_key_type'],[id$='_validator_type'],[name$='[omit]'],[name$='[hidden_value]']");
+  var fields_to_disable = fields.find("[name$='[required]'],[id$='_key_type'],[id$='_validator_type'],[name$='[omit]'],[name$='[hidden_value]'],[name$='[parameter_type]']");
```

Comment:
@xprazak2,
Sorry, just one small inline change.

Here, it is already mentioned `[id$='_key_type']` so either you can make it `[id$='_parameter_type']` or  remove `[id$='_key_type']` and keep [name$='[parameter_type]'].